modules/ROOT/pages/telemetry-exporter.adoc

```diff
@@ -0,0 +1,146 @@
+= Export Anypoint Platform Data to Third-Party Apps
+
+The Telemetry Exporter feature enables you to export xref:access-management::audit-logging.adoc[_audit logs_] to third-party observability and analytics apps, such as Azure Monitor, Datadog, and Splunk HEC.
+//trace data should be a link once the topic exists
+
+To use the Telemetry Exporter feature, you must create a _connection_ with a third-party app to authorize Anypoint Platform to share data. You must then create a _configuration_ to specify which business groups allow content to be exported to connections. You can also edit and delete connections and configurations. 
+
+include::partial$telemetry-exporter-notes.adoc[tag=hourDelay]
+
+// add back to configs: "configuration to specify the types of data Anypoint Platform exports as well as..."
+
+== Supported Third-Party Apps
+
+The Telemetry Exporter feature enables you to create connections with most third-party observability and analytics apps that have OpenTelemetry-supported endpoints and allow you to create an instrumentation key or access token. Azure Monitor and Splunk HEC are also compatible. 
+
+== Before You Begin
+
+Before you configure the Telemetry Exporter feature, ensure that you have:
+
+* To create or modify connections, an Anypoint Platform account with the Telemetry Exporter Administrator permission
```

Comment:
This does not seem to follow the lead-in.

Reply:
```suggestion
* Telemetry Exporter Administrator permission, so you can create or modify connections
```